src/api/v3/pages.py

```diff
@@ -29,17 +29,19 @@ def transform_page(page_translation):
             "id": page_translation.page.parent.id,
```

Comment:
Until now the API returned a `0` if there is no page parent. With Django, the response currently contains `null`. We need to change this as well.